app/src/pages/admin/pluginsPage/pluginsToolbar/actionPanel/actionPanel.scss

```diff
@@ -0,0 +1,11 @@
+.action-buttons {
+  display: flex;
+  justify-content: flex-end;
+  .action-button {
+    margin-left: 15px;
+    svg {
```

Comment:
Is it necessary? Button with the same icon is located on Launches page ("Import") and it doesn't have such style.